app/assets/stylesheets/_variables.scss

```diff
@@ -5,37 +5,39 @@
 //=================================================================================
 
 // COLOR PALETTE
+// See US federal standards at: https://standards.usa.gov/colors/
 
 // Greyscale.
-$black: rgb(0,0,0);
+$black: #212121; // "base"
```

Comment:
So usually [especially if I didn't know you personally] if I had questions or comments or concerns about your code, this is where I could make comments, start a review, close your pull request [aka refuse it], etc. Since we're talking on slack we're bypassing this, but know that I can usually review your code at this step.